### ISSUE TITLE
"Update Revision on Launch" not working if initial project is missing

### ISSUE THREAD

**Issue by yw5vruox7dr** [ansible/awx#11884]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Summary

When using AWX in a dynamic environment like Kubernetes, it can happen that the awx instance is rescheduled to another k8s worker node. With that node switch, the initial clone of projects will be lost. All tasks will fail until someone manual triggers project syncs. 

The option "Update Revision on Launch" in the project settings is not honored.

### AWX version

20.0.0

### Select the relevant components

- [ ] UI
- [x] API
- [ ] Docs

### Installation method

kubernetes

### Modifications

no

### Steps to reproduce

1. Create awx instance via awx operator
2. Create & sync project
3. Create template
4. Check if template is working
5. Re-deploy awx instance (delete & apply)

### Expected results

Template job is normal running

### Actual results

Template job is not starting and displays an error:
```
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/tasks/jobs.py", line 449, in run
    self.pre_run_hook(self.instance, private_data_dir)
  File "/var/lib/awx/venv/awx/lib64/python3.9/site-packages/awx/main/tasks/jobs.py", line 892, in pre_run_hook
    raise RuntimeError(msg)
RuntimeError: The project revision for this job template is unknown due to a failed update.
```

### Additional information

Similar issue like in https://github.com/ansible/awx/issues/8571

**Comment by AlanCoding** [comment]
This error message "The project revision for this job template is unknown due to a failed update" is referencing the fact that a project update (triggered by `update_on_launch` in your case it looks like) failed. To know what happened, we need to look at the output of that project update.

**Comment by yw5vruox7dr** [comment]
@AlanCoding there is no output in the source control job.

job overview: https://i.imgur.com/T9Q74az.png
template job: https://i.imgur.com/dqzDIhh.png
source control job: https://i.imgur.com/D4h8V1Z.png

Between 7,8 and 9,10 I redeployed the awx instance.

**Comment by AlanCoding** [comment]
Of course the _ideal_ is that the UI will find any relevant information, but it may not be doing this. In the `/api/v2/project_updates/N/` details, critical bits of information are often `status`, `job_explanation`, and `result_traceback`. If none of these are helpful, then there might be something that went _really_ wrong in the logs.

**Comment by yw5vruox7dr** [comment]
No more useful information in these logs:

<details>
  <summary>API ouput for project update (Click to expand)</summary>
  
  ```
{
    "id": 6,
    "type": "project_update",
    "url": "/api/v2/project_updates/6/",
    "related": {
        "credential": "/api/v2/credentials/4/",
        "unified_job_template": "/api/v2/projects/8/",
        "stdout": "/api/v2/project_updates/6/stdout/",
        "project": "/api/v2/projects/8/",
        "cancel": "/api/v2/project_updates/6/cancel/",
        "scm_inventory_updates": "/api/v2/project_updates/6/scm_inventory_updates/",
        "notifications": "/api/v2/project_updates/6/notifications/",
        "events": "/api/v2/project_updates/6/events/"
    },
    "summary_fields": {
        "organization": {
            "id": 1,
            "name": "Default",
            "description": ""
        },
        "project": {
            "id": 8,
            "name": "test-project",
            "description": "",
            "status": "failed",
            "scm_type": "git",
            "allow_override": false
        },
        "credential": {
            "id": 4,
            "name": "test-cred",
            "description": "",
            "kind": "scm",
            "cloud": false,
            "kubernetes": false,
            "credential_type_id": 2
        },
        "unified_job_template": {
            "id": 8,
            "name": "test-project",
            "description": "",
            "unified_job_type": "project_update"
        },
        "instance_group": {
            "id": 1,
            "name": "controlplane",
            "is_container_group": false
        },
        "user_capabilities": {
            "delete": true,
            "start": true
        }
    },
    "created": "2022-03-18T19:38:40.275111Z",
    "modified": "2022-03-18T19:38:42.677612Z",
    "name": "test-project",
    "description": "",
    "local_path": "_8__test_project",
    "scm_type": "git",
    "scm_url": "xxxx_removed_for_privacy_xxx",
    "scm_branch": "",
    "scm_refspec": "",
    "scm_clean": true,
    "scm_track_submodules": false,
    "scm_delete_on_update": false,
    "credential": 4,
    "timeout": 0,
    "scm_revision": "",
    "unified_job_template": 8,
    "launch_type": "dependency",
    "status": "failed",
    "execution_environment": null,
    "failed": true,
    "started": null,
    "finished": "2022-03-18T19:38:51.652696Z",
    "canceled_on": null,
    "elapsed": 0.0,
    "job_args": "",
    "job_cwd": "",
    "job_env": {},
    "job_explanation": "Task was marked as running but was not present in the job queue, so it has been marked as failed.",
    "execution_node": "awx-test-f9bfcd9cf-sbhzb",
    "result_traceback": "",
    "event_processing_finished": true,
    "launched_by": {
        "id": null,
        "name": "Generated by AWX",
        "type": "Dependency",
        "url": null
    },
    "work_unit_id": null,
    "project": 8,
    "job_type": "check",
    "job_tags": "update_git,install_roles,install_collections",
    "host_status_counts": {},
    "playbook_counts": {
        "play_count": 0,
        "task_count": 0
    }
}
  ```
</details>

```
    "status": "failed",
    "job_explanation": "Task was marked as running but was not present in the job queue, so it has been marked as failed.",
    "result_traceback": "",
```

<details>
  <summary>Logs of task container (Click to expand)</summary>
  
  ```
  2022-03-18 19:38:41,611 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.models.inventory Going to update inventory computed fields, pk=1
2022-03-18 19:38:41,688 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.models.inventory Finished updating inventory computed fields, pk=1, in 0.076 seconds
2022-03-18 19:38:42,142 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.dispatch task 08423575-86c3-4405-99c0-afdce7e17d08 starting awx.main.scheduler.tasks.run_task_manager(*[])
2022-03-18 19:38:42,144 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler Running task manager.
2022-03-18 19:38:42,183 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler Starting Scheduler
2022-03-18 19:38:42,414 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle job-5 acknowledged
2022-03-18 19:38:42,522 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle projectupdate-6 created
2022-03-18 19:38:42,588 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler Spawned project_update 6 (pending) as dependency of job 5 (pending)
2022-03-18 19:38:42,648 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler Starting project_update 6 (pending) in group controlplane instance awx-test-f9bfcd9cf-sbhzb (remaining_capacity=678)
2022-03-18 19:38:42,664 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle projectupdate-6 execution node chosen
2022-03-18 19:38:42,665 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle projectupdate-6 controller node chosen
2022-03-18 19:38:42,665 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler Submitting job project_update 6 (waiting) to queue awx-test-f9bfcd9cf-sbhzb controlled by awx-test-f9bfcd9cf-sbhzb.
2022-03-18 19:38:42,714 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle projectupdate-6 waiting
2022-03-18 19:38:42,714 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler project_update 6 (waiting) consumed 1 capacity units from controlplane with prior total of 0
2022-03-18 19:38:42,717 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle job-5 blocked by projectupdate-6
2022-03-18 19:38:42,717 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.main.scheduler Finishing Scheduler
2022-03-18 19:38:51,086 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task 7f57e697-c692-4624-92f2-75f064ea81ae starting awx.main.tasks.cluster_node_heartbeat(*[])
2022-03-18 19:38:51,088 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Cluster node heartbeat task.
2022-03-18 19:38:51,127 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task ca23a5b7-b3f4-45fa-89e3-5c56723e33ee starting awx.main.tasks.awx_k8s_reaper(*[])
2022-03-18 19:38:51,141 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task f176b41c-01cc-424d-97df-38eccae3226d starting awx.main.tasks.awx_receptor_workunit_reaper(*[])
2022-03-18 19:38:51,144 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Checking for unreleased receptor work units
2022-03-18 19:38:51,219 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task 8a85894e-3e1f-416a-b3a3-cd97f3be5836 starting awx.main.tasks.awx_periodic_scheduler(*[])
2022-03-18 19:38:51,222 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Checking for orphaned k8s pods for default-2.
2022-03-18 19:38:51,310 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Starting periodic scheduler
2022-03-18 19:38:51,310 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task 4feaa461-7dcd-4920-ba75-164af767eb6a starting awx.main.scheduler.tasks.run_task_manager(*[])
2022-03-18 19:38:51,311 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.scheduler Running task manager.
2022-03-18 19:38:51,318 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Last scheduler run was: 2022-03-18 19:38:20.944020+00:00
2022-03-18 19:38:51,400 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.scheduler Starting Scheduler
2022-03-18 19:38:51,401 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task 6ff31384-7e13-41b6-b0c1-0ecb55741fc2 starting awx.main.analytics.analytics_tasks.send_subsystem_metrics(*[])
2022-03-18 19:38:51,643 ERROR    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.scheduler awx-test-f9bfcd9cf-sbhzb is not a registered instance; reaping project_update 6 (waiting)
2022-03-18 19:38:51,879 ERROR    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch project_update 6 (failed) is no longer running; reaping
2022-03-18 19:38:51,908 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.analytics.job_lifecycle job-5 blocked by projectupdate-6
2022-03-18 19:38:51,909 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.scheduler Finishing Scheduler
2022-03-18 19:38:52,080 INFO     [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Host awx-test-f9bfcd9cf-sbhzb Automatically Deprovisioned.
2022-03-18 19:38:52,081 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.dispatch task fded04ad-f8a6-494c-8826-725c60dd3c7c starting awx.main.tasks.apply_cluster_membership_policies(*[])
2022-03-18 19:38:52,129 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Waited 0.046567678451538086 seconds to obtain lock name: cluster_policy_lock
2022-03-18 19:38:52,163 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Total instances: 1, available for policy: 1
2022-03-18 19:38:52,165 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Policy percentage, adding Instances [2] to Group controlplane
2022-03-18 19:38:52,166 DEBUG    [68666e4ab5864b8ab3eb5afed261f3b6] awx.main.tasks Cluster policy no-op finished in 0.0353703498840332 seconds
2022-03-18 19:38:54,203 DEBUG    [-] awx.main.commands.run_callback_receiver 59 is alive
2022-03-18 19:38:54,203 DEBUG    [-] awx.main.commands.run_callback_receiver 59 is alive
RESULT 2
OKREADY
2022-03-18 19:39:11,447 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task 19a7590e-5e95-4ab8-8d20-8a9b5f725b33 starting awx.main.scheduler.tasks.run_task_manager(*[])
2022-03-18 19:39:11,448 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Running task manager.
2022-03-18 19:39:11,490 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task c049300e-d3e4-4db2-b6db-f395c2428618 starting awx.main.analytics.analytics_tasks.send_subsystem_metrics(*[])
2022-03-18 19:39:11,491 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Starting Scheduler
2022-03-18 19:39:11,897 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.analytics.job_lifecycle job-5 controller node chosen
2022-03-18 19:39:11,990 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.analytics.job_lifecycle job-5 waiting
2022-03-18 19:39:11,990 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler job 5 (waiting) consumed 2 capacity units from default with prior total of 0
2022-03-18 19:39:11,991 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Finishing Scheduler
2022-03-18 19:39:12,052 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task af289385-e913-4a57-bf68-43080abd926b starting awx.main.tasks.RunJob(*[5])
2022-03-18 19:39:12,501 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.analytics.job_lifecycle job-5 pre run
2022-03-18 19:39:12,611 ERROR    [b0d491d7c81b44e899672d4581788eda] awx.main.tasks job 5 (running) Exception occurred while running task
Traceback (most recent call last):
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/awx/main/tasks.py", line 1442, in run
    self.pre_run_hook(self.instance, private_data_dir)
  File "/var/lib/awx/venv/awx/lib64/python3.8/site-packages/awx/main/tasks.py", line 1874, in pre_run_hook
    raise RuntimeError(msg)
RuntimeError: The project revision for this job template is unknown due to a failed update.
2022-03-18 19:39:12,612 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.tasks job 5 (running) finished running, producing 0 events.
2022-03-18 19:39:12,613 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.analytics.job_lifecycle job-5 post run
2022-03-18 19:39:12,731 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.analytics.job_lifecycle job-5 finalize run
2022-03-18 19:39:12,772 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task 1c6dc208-0689-4a6c-aca0-bb836ddd118e starting awx.main.tasks.update_inventory_computed_fields(*[1])
2022-03-18 19:39:12,777 WARNING  [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch job 5 (error) encountered an error (rc=None), please see task stdout for details.
2022-03-18 19:39:12,778 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task af289385-e913-4a57-bf68-43080abd926b starting awx.main.tasks.handle_work_error(*['af289385-e913-4a57-bf68-43080abd926b'])
2022-03-18 19:39:12,779 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.tasks Executing error task id af289385-e913-4a57-bf68-43080abd926b, subtasks: [{'type': 'job', 'id': 5}]
2022-03-18 19:39:12,820 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task af289385-e913-4a57-bf68-43080abd926b starting awx.main.tasks.handle_work_success(*[])
2022-03-18 19:39:12,821 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task e6d7f8a6-441e-4209-a7a4-45186fc86b32 starting awx.main.scheduler.tasks.run_task_manager(*[])
2022-03-18 19:39:12,822 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Running task manager.
2022-03-18 19:39:12,836 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.models.inventory Going to update inventory computed fields, pk=1
2022-03-18 19:39:12,912 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.dispatch task 12671c3c-a302-4354-8716-c02685d348ab starting awx.main.scheduler.tasks.run_task_manager(*[])
2022-03-18 19:39:12,913 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Running task manager.
2022-03-18 19:39:12,914 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Starting Scheduler
2022-03-18 19:39:13,008 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Not running scheduler, another task holds lock
2022-03-18 19:39:13,022 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.models.inventory Finished updating inventory computed fields, pk=1, in 0.185 seconds
2022-03-18 19:39:13,146 DEBUG    [b0d491d7c81b44e899672d4581788eda] awx.main.scheduler Finishing Scheduler
  ```
  </details>
  
  <details>
  <summary>Logs of web container (Click to expand)</summary>
  
  ```
  2022-03-18 19:38:40,883 DEBUG    [3776cc8f82d2483582e013550cc14b19] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/job_templates/9/launch/'>, response_time: 0.345s
[pid: 39|app: 0|req: 5/22] 127.0.0.1 () {58 vars in 2529 bytes} [Fri Mar 18 19:38:40 2022] GET /api/v2/job_templates/9/launch/ => generated 754 bytes in 347 msecs (HTTP/1.1 200) 14 headers in 583 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:40 +0000] "GET /api/v2/job_templates/9/launch/ HTTP/1.1" 200 754 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:40,894 DEBUG    [437eedad24774caab797e7df96385a20] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/job_templates/9/survey_spec/'>, response_time: 0.345s
[pid: 40|app: 0|req: 6/23] 127.0.0.1 () {58 vars in 2539 bytes} [Fri Mar 18 19:38:40 2022] GET /api/v2/job_templates/9/survey_spec/ => generated 2 bytes in 346 msecs (HTTP/1.1 200) 14 headers in 589 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:40 +0000] "GET /api/v2/job_templates/9/survey_spec/ HTTP/1.1" 200 2 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:41,346 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.job_lifecycle job-5 created
2022-03-18 19:38:42,159 DEBUG    [00c02ef8933a439388588919e8e86a11] awx.analytics.performance request: <WSGIRequest: POST '/api/v2/job_templates/9/launch/'>, response_time: 1.204s
[pid: 37|app: 0|req: 5/24] 127.0.0.1 () {64 vars in 2644 bytes} [Fri Mar 18 19:38:40 2022] POST /api/v2/job_templates/9/launch/ => generated 2732 bytes in 1205 msecs (HTTP/1.1 201) 15 headers in 616 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:42 +0000] "POST /api/v2/job_templates/9/launch/ HTTP/1.1" 201 2732 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:42,694 DEBUG    [9caf215a3198489c88a19c8fd698c0f6] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_jobs/?id=5'>, response_time: 0.421s
[pid: 39|app: 0|req: 6/25] 127.0.0.1 () {58 vars in 2518 bytes} [Fri Mar 18 19:38:42 2022] GET /api/v2/unified_jobs/?id=5 => generated 2660 bytes in 423 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:42 +0000] "GET /api/v2/unified_jobs/?id=5 HTTP/1.1" 200 2660 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:43,317 DEBUG    [ec6abcf39a47493091064176d7f68614] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_jobs/?id=5'>, response_time: 0.577s
[pid: 39|app: 0|req: 7/26] 127.0.0.1 () {58 vars in 2518 bytes} [Fri Mar 18 19:38:42 2022] GET /api/v2/unified_jobs/?id=5 => generated 2657 bytes in 579 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:43 +0000] "GET /api/v2/unified_jobs/?id=5 HTTP/1.1" 200 2657 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:43,782 DEBUG    [f20cbfeac1c94d1295fa9b6f804cd44c] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_jobs/?id=5'>, response_time: 0.403s
127.0.0.1 - - [18/Mar/2022:19:38:43 +0000] "GET /api/v2/unified_jobs/?id=5 HTTP/1.1" 200 2657 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
[pid: 38|app: 0|req: 5/27] 127.0.0.1 () {58 vars in 2518 bytes} [Fri Mar 18 19:38:43 2022] GET /api/v2/unified_jobs/?id=5 => generated 2657 bytes in 404 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
2022-03-18 19:38:44,292 DEBUG    [e9c829ebedb641cdb20a945190f363a1] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_jobs/?id=5'>, response_time: 0.434s
[pid: 39|app: 0|req: 8/28] 127.0.0.1 () {58 vars in 2518 bytes} [Fri Mar 18 19:38:43 2022] GET /api/v2/unified_jobs/?id=5 => generated 2657 bytes in 435 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:44 +0000] "GET /api/v2/unified_jobs/?id=5 HTTP/1.1" 200 2657 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:44,798 WARNING  [-] awx.main.wsbroadcast Connection from awx-test-f9bfcd9cf-xqm8d to 10.233.80.65 timed out.
2022-03-18 19:38:44,809 DEBUG    [-] awx.main.wsbroadcast Connection from awx-test-f9bfcd9cf-xqm8d to 10.233.80.65 attempt number 3.
2022-03-18 19:38:44,811 DEBUG    [d55da4f21b0a4e34bece91349fd9280d] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/unified_jobs/?id=5'>, response_time: 0.418s
[pid: 38|app: 0|req: 6/29] 127.0.0.1 () {58 vars in 2518 bytes} [Fri Mar 18 19:38:44 2022] GET /api/v2/unified_jobs/?id=5 => generated 2657 bytes in 420 msecs (HTTP/1.1 200) 14 headers in 578 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:44 +0000] "GET /api/v2/unified_jobs/?id=5 HTTP/1.1" 200 2657 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:45,399 DEBUG    [e068ba28329542da83e8cdbbf75b9413] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/jobs/5/'>, response_time: 0.418s
[pid: 40|app: 0|req: 7/30] 127.0.0.1 () {58 vars in 2497 bytes} [Fri Mar 18 19:38:44 2022] GET /api/v2/jobs/5/ => generated 2800 bytes in 420 msecs (HTTP/1.1 200) 14 headers in 586 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:45 +0000] "GET /api/v2/jobs/5/ HTTP/1.1" 200 2800 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:45,625 DEBUG    [0f9d4c058d184b509637a5ec59deabe2] awx.analytics.performance request: <WSGIRequest: OPTIONS '/api/v2/jobs/5/job_events/'>, response_time: 0.155s
127.0.0.1 - - [18/Mar/2022:19:38:45 +0000] "OPTIONS /api/v2/jobs/5/job_events/ HTTP/1.1" 200 11910 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
[pid: 37|app: 0|req: 6/31] 127.0.0.1 () {60 vars in 2559 bytes} [Fri Mar 18 19:38:45 2022] OPTIONS /api/v2/jobs/5/job_events/ => generated 11910 bytes in 156 msecs (HTTP/1.1 200) 15 headers in 602 bytes (1 switches on core 0)
2022-03-18 19:38:46,006 DEBUG    [97a59a6c2a69416e95cd7d5dfd681537] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/jobs/5/job_events/?counter__gte=1&limit=50&order_by=counter'>, response_time: 0.190s
127.0.0.1 - - [18/Mar/2022:19:38:46 +0000] "GET /api/v2/jobs/5/job_events/?counter__gte=1&limit=50&order_by=counter HTTP/1.1" 200 14 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
[pid: 39|app: 0|req: 9/32] 127.0.0.1 () {58 vars in 2600 bytes} [Fri Mar 18 19:38:45 2022] GET /api/v2/jobs/5/job_events/?counter__gte=1&limit=50&order_by=counter => generated 14 bytes in 193 msecs (HTTP/1.1 200) 15 headers in 599 bytes (1 switches on core 0)
2022-03-18 19:38:46,052 DEBUG    [307ff69703484554a00c4a3b01f4cfdb] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/jobs/5/job_events/?limit=1&order_by=-counter'>, response_time: 0.192s
[pid: 38|app: 0|req: 7/33] 127.0.0.1 () {58 vars in 2570 bytes} [Fri Mar 18 19:38:45 2022] GET /api/v2/jobs/5/job_events/?limit=1&order_by=-counter => generated 14 bytes in 194 msecs (HTTP/1.1 200) 15 headers in 599 bytes (1 switches on core 0)
127.0.0.1 - - [18/Mar/2022:19:38:46 +0000] "GET /api/v2/jobs/5/job_events/?limit=1&order_by=-counter HTTP/1.1" 200 14 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
2022-03-18 19:38:51,763 WARNING  [-] awx.main.wsbroadcast Removing {'awx-test-f9bfcd9cf-sbhzb'} from websocket broadcast list
2022-03-18 19:38:51,765 WARNING  [-] awx.main.wsbroadcast Connection from awx-test-f9bfcd9cf-xqm8d to 10.233.80.65 cancelled.
RESULT 2
OKREADY
2022-03-18 19:39:13,300 DEBUG    [da8364b5b561480293ae0b9213b6de59] awx.analytics.job_lifecycle job-5 event processing finished
2022-03-18 19:39:13,342 DEBUG    [da8364b5b561480293ae0b9213b6de59] awx.analytics.performance request: <WSGIRequest: GET '/api/v2/jobs/5/'>, response_time: 0.529s
127.0.0.1 - - [18/Mar/2022:19:39:13 +0000] "GET /api/v2/jobs/5/ HTTP/1.1" 200 3511 "http://127.0.0.1:8080/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.136 Safari/537.36" "-"
[pid: 38|app: 0|req: 8/34] 127.0.0.1 () {58 vars in 2497 bytes} [Fri Mar 18 19:39:12 2022] GET /api/v2/jobs/5/ => generated 3511 bytes in 530 msecs (HTTP/1.1 200) 14 headers in 586 bytes (1 switches on core 0)
  ```
  </details>

**Comment by yw5vruox7dr** [comment]
It is working if you waiting long enough after a redeployment (I waited ~30 minutes, maybe the actual wait time is shorter). Probably a internal awx job that needs to run before you really can use awx.

Update: You have to wait at least 1:20 minutes before you should run any jobs.
